src/load-data.test.js

```diff
@@ -5,22 +5,45 @@ import sinon from 'sinon'
 
 import loadDataIntoLocalStorage from './load-data'
 
+const DEFAULT_SERIALIZED_DATA = { segmentsData: {}, since: 0, splitsData: {}, usingSegmentsCount: 0 }
 const SMALLER_SINCE = 0
 const LARGER_SINCE = 1
 
+function expectAllStubsToNotBeCalled (localStorageOverride) {
```

Comment:
Since this test file calls this function once, I'd suggest you inline the `expect`s.  It makes the test that calls it easier to read/understand.